adev/src/app/sub-navigation-data.ts

```diff
@@ -1360,6 +1360,11 @@ const REFERENCE_SUB_NAVIGATION_DATA: NavigationItem[] = [
         path: 'reference/migrations/self-closing-tags',
         contentPath: 'reference/migrations/self-closing-tags',
       },
+      {
+        label: 'NgClass to Class',
+        path: 'reference/migrations/ngclass-to-class',
+        contentPath: 'reference/migrations/ngclass-to-class',
+      },
```

Comment:
```suggestion
      {
        label: 'NgClass to Class',
        path: 'reference/migrations/ngclass-to-class',
        contentPath: 'reference/migrations/ngclass-to-class',
        status: 'new'
      },
```
So we have that fancy new badge 😄

Reply:
wao :)